src/templates/ja/2019/chapter_author.html

```diff
@@ -0,0 +1,24 @@
+{% for author in authors %}
```

Comment:
No need to update Japanese templates, we can leave that for the final translation step.